test/unit/processing/settlement/schema.test.js

```diff
@@ -0,0 +1,116 @@
+const schema = require('../../../../app/processing/settlement/schema')
```

Comment:
well picked up -- something to add to the standardise 'steps' tech debt ticket: https://eaflood.atlassian.net/browse/PAY-2406

Reply:
Yes there's quite a few code branches not covered with tests or areas which may benefit from a few extra targeted tests.

Didn't want to spend too much time fixing all that in this PR, particularly as we had this one, but any obvious benefit I could add within the scope of what I was adding I tried to do.